Source/Meadow.Units/ReactivePower.cs

```diff
@@ -0,0 +1,412 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics.Contracts;
+using System.Runtime.InteropServices;
+using Meadow.Units.Conversions;
+
+namespace Meadow.Units
+{
+    /// <summary>
+    /// Represents Reactive Power
+    /// </summary>
+    [Serializable]
+    [ImmutableObject(true)]
+    [StructLayout(LayoutKind.Sequential)]
+    public struct ReactivePower :
+        IComparable, IFormattable, IConvertible,
+        IEquatable<double>, IComparable<double>
+    {
+        /// <summary>
+        /// Creates a new `ReactivePower` object.
+        /// </summary>
+        /// <param name="value">The ReactivePower value.</param>
+        /// <param name="type">kilometers meters per second by default.</param>
+        public ReactivePower(double value, UnitType type = UnitType.VoltAmpheres)
+        {
+            Value = ReactivePowerConversions.Convert(value, type, UnitType.VoltAmpheres);
+        }
+
+        /// <summary>
+        /// Creates a new `ReactivePower` object from an existingReactivePower object
+        /// </summary>
+        /// <param name="reactivePower"></param>
+        public ReactivePower(ReactivePower reactivePower)
+        {
+            this.Value = reactivePower.Value;
+        }
+
+        /// <summary>
+        /// Internal canonical value.
+        /// </summary>
+        private readonly double Value;
+
+        /// <summary>
+        /// The type of units available to describe ReactivePower.
+        /// </summary>
+        public enum UnitType
```

Comment:
This is interesting, because reactive power units are actually `XAmperesReactive` i.e. `VoltAmperesReactive`. I would consider either renaming, or just reusing the `ApparentPower` type.

Reply:
Also, if you're doing `ApparentPower` and `ReactivePower`, do we already have a `RealPower` or `Power` unit that is measured in `Watts?`

Reply:
We do have Power already with Watts. 

And agreed, I don't love the duplication between Reactive and Apparent Power. But I feel like it's incorrect to return a Reactive Power unit for Apparent Power (or vise-versa). 

I also debated just adding the reactive/apparent power units to Power but the industry seems to keep them separate. And that might also imply a conversion from Reactive Power to Real Power.

Reply:
After thinking about it I'm personally not in favor of combining. These are non-SI units and I think the unit names need to be specific. But if people that know better have other options I'm of course happy to change.

Reply:
@ctacke